tsl/test/sql/remote_txn.sql

```diff
@@ -217,7 +217,11 @@ FROM _timescaledb_internal.show_connection_cache() ORDER BY 1,4;
 BEGIN;
     SELECT remote_node_killer_set_event('pre-prepare-transaction', 'loopback');
     SELECT test.remote_exec('{loopback}', $$ INSERT INTO "S 1"."T 1" VALUES (10002,1,'bleh', '2001-01-01', '2001-01-01', 'bleh') $$);
+-- since the error message might differ between environments we suppress normal output and print sqlstate
+--<exclude_from_test>
 COMMIT;
+--</exclude_from_test>
+select :SQLSTATE AS sqlstate;
```

Comment:
Where do we set the `SQLSTATE` in our code?

Reply:
> Where do we set the `SQLSTATE` in our code?

It's psql feature: https://www.postgresql.org/docs/current/app-psql.html

Reply:
There is also VERBOSITY sqlstate for psql but unfortunately thats PG12 only

Reply:
Ok, cool for some reason I wasn't able to query it as is psql app, so thought it is something else.